docs/source/using/profiles.adoc

```diff
@@ -63,4 +111,36 @@ To delete a profile, run:
 $ minishift profile delete PROFILE_NAME
 ----
 
-NOTE: The default profile 'minishift'  can not be deleted.
+NOTE: The default profile *minishift* cannot be deleted.
+
+[[setting-global-config-profiles]]
+== Setting Global Configuration for Profiles
+
+Currently, you cannot set any global configuration for profiles.
+If you want to apply a common configuration to all your profiles then you need to apply the settings for each profile separately.
+See xref:../using/profiles.adoc#setting-global-config-profiles[Setting Global Configuration For Profiles].
```

Comment:
You realise you are referring to yourself here. 

In the intro you say: "If you have common configuration options you want to apply to all profiles, you have to do this manually per profile See Setting Global Configuration For Profiles."

One here we offer no solution and even reference to the very section we are reading.

Reply:
I think we need to be more explicit and give an example for the common env use case. For example the CPU count and memory size is probably something people want to have for all profiles. How can this be done.